.circleci/config.yml

```diff
@@ -25,6 +25,5 @@ workflows:
           # <regex path-to-test> <parameter-to-set> <value-of-pipeline-parameter>
           mapping: |
             presto-native-execution/.* run_native_specific_jobs true
-            presto-docs/.* run_linux_tests false
```

Comment:
Possibly we could change the default to false, and then use a regexp that matches everything except presto-docs to set run_linux_tests to true.